en/SetupView/Sensors.md

```diff
@@ -1,42 +1,118 @@
+
+<img src="../../images/setup/sensor_orientation_sensor_settings.jpg" style="width: 600px;"/>
+
+<img src="../../images/setup/sensor_orientation_set_orientations.jpg" style="width: 600px;"/>
+
+> **Note** The orientation is specified as YAW, PITCH and/or ROLL offsets relative to the forward-facing-upright orientation (clock-wise rotation around the Z, Y and X axis, respectively).
+>
+> <img src="../../images/setup/flight_controller_orientation.png" style="width: 600px;"/>
+>
+> For example, the image below shows some `ROTATION_YAW_XX` orientations:
+> <img src="../../images/setup/flight_controller_yaw_rotation.png" style="width: 600px;"/>
 
-For ArduPilot follow text textual instructions in the center display, clicking the Next button to capture each position.
 
-For PX4 you will be guided with a visual display of positions.
 
 ## Compass
 
-### ArduPilot (newer firmwares)
-For newer ArduPilot firmwares compass calibration will be done using the Onboard calibration support which provides for more accurate calibration. You need to rotate the vehicle randomly around all axes until the progress bar fills all the way to the right and the calibration completes. When the calibration completes you will get the following results:
+### PX4 Compass Calibration
+
+For PX4 you will be guided to position the vehicle in a number of set orientations and rotate the vehicle about the specified axis. 
+
+> **Note** This process is also used for calibrating the compass on older ArduPilot firmware.
+
+The calibration steps are:
+
+1. Click the **Compass** sensor button
+
+1. Click **OK** to start the calibration. 
+
+   <img src="../../images/setup/sensor_compass_orientation_prompt.jpg" width="250px" title="Start calibration prompt" />
+
+   > **Note** You should already have set the [Flight Controller Orientation](#flight_controller_orientation) in the previous section. If not, you can also set it here.
+
+1. Place the vehicle in any of the orientations shown in red (incomplete) and hold it still. Once prompted (the orientation-image turns yellow) rotate the vehicle around the specified axis in either/both directions. Once the calibration is complete in that orientation the associated image on the screen will turn green.
+ 
+   ![](../../images/setup/sensors_px4_compass.jpg)
+   
+1. Repeat the calibration process for all vehicle orientations.
+   
+Once you've rotated the vehicle in all the positions *QGroundControl* will display *Calibration complete* (all orientation images will be displayed in green and the progress bar will fill completely). You can then proceed to the next sensor. 
+
+
+### ArduPilot Compass Calibration
 
-![](../../images/setup/ArudPilolCompassCalOnboardResult.jpg)
+ArduPilot uses onboard calibration support that allows for more accurate calibration. 
```

Comment:
I would have liked to verify/expand on this but my hardware doesn't work. Could this be tested using a MockLink?